src/databricks/labs/remorph/resources/reconcile/dashboards/reconciliation_metrics/05_0_summary_table.sql

```diff
@@ -0,0 +1,32 @@
+/* --title 'Summary Table' --width 6 --height 6 */
+SELECT main.recon_id,
+       main.source_type,
+       main.report_type,
+       main.source_table.`catalog` AS source_catalog,
+       main.source_table.`schema` AS source_schema,
+       main.source_table.table_name AS source_table_name,
+       CASE
+           WHEN COALESCE(main.source_table.catalog, '') <> '' THEN CONCAT(main.source_table.catalog, '.', main.source_table.schema, '.', main.source_table.table_name)
```

Comment:
This case statement can be converted as below, avoiding Unnecessary use of `Coalesece`
```
 IF(
                     ISNULL(source_catalog),
                     CONCAT_WS('.', source_schema, source_table_name),
                     CONCAT_WS(
                             '.',
                             source_catalog,
                             source_schema,
                             source_table_name
                     )
             ) AS source_table,
```